code/src/java/pcgen/gui2/kits/KitPanel.java

```diff
@@ -283,12 +283,14 @@ public KitTreeViewModel(CharacterFacade character, boolean isAvailModel)
 				ListFacade<KitFacade> kitList = new DefaultListFacade<KitFacade>(character.getAvailableKits());
 				kits.setDelegate(kitList);
 				character.getKits().addListListener(this);
-				columns = Arrays.asList(new DefaultDataViewColumn("in_source", String.class, false)); //$NON-NLS-1$
+				columns = Arrays.asList(new DefaultDataViewColumn("in_descrip", String.class), //$NON-NLS-1$
```

Comment:
Ah, you are right. Kits are rather devoid of descriptions so I'll make sure that column is hidden by default. Nice catch!